test/test_regression_cli.py

```diff
@@ -54,17 +48,17 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    os.chdir(rood_dir)
+    os.system('cp -r {0} {1}'.format(os.path.join(rood_dir, "test/raw_data"), rood_dir))
 
 
 def teardown_module():
     """Cleanup temporary output files after testing and return to root directory"""
-    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    os.chdir(rood_dir)
```

Comment:
Are we changing the working directory anywhere to anything other than the root directory anywhere in the tests now? If not then we shouldn't need all of this directory changing code. Your totally right that in the old setup we were moving into `test` and then back out of it in some cases, so if we changed in and then failed, we might not have gotten changed out. I'm just wondering if that sort of issue is still possible.